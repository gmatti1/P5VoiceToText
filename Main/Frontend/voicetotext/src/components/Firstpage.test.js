import React from 'react';

import {render, cleanup} from '@testing-library/react';
import FirstPage from './Firstpage';

/**
 * 
 *
 * @version 1.0
 * @author [Shashidhar Reddy Vanteru]
 * @copyright [Copyright 2020, P5VoiceToText]
 * @credits  [Shashidhar Reddy Vanteru]
 * @email "svanter1@asu.edu"
 * 
 */

afterEach(cleanup)
it('should take a snapshot', () => {
  const { asFragment } = render(<FirstPage />)
  
  expect(asFragment(<FirstPage />)).toMatchSnapshot()
 });

