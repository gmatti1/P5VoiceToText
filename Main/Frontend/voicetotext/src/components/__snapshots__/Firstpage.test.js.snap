// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="Firstpage"
  >
    <div
      class="Navbarsize"
    >
      <nav
        class="navbar sticky-top justify-content-center"
        id="navbg"
      >
        <img
          alt="Website logo"
          class="App-logobig"
          src="pchlogo.png"
        />
      </nav>
    </div>
    <div
      class="App-bg"
    />
    <header
      class="Appheaderfp"
    >
      <h1
        class="Apptitlefp"
      >
        Voice-To-Text Application of Pediatric IMIST-AMBO Criteria
      </h1>
    </header>
    <header
      class="App-footer"
    >
      <a>
        <button>
          <img
            alt="arrow"
            class="Arrow"
            src="arrow.png"
          />
        </button>
      </a>
    </header>
  </div>
</DocumentFragment>
`;
